bigquery2/bigquery_endpoint.bal

```diff
@@ -425,7 +434,59 @@ remote function Client.getQueryResults(string projectId, string jobId, string ne
     }
 }
 
-function Client.init(BigqueryConfiguration bigqueryConfig) {
+public remote function Client.getAccessTokenFromServiceAccount(string keyStoreLocation, string serviceAccount,
+                                                               string scope) returns @tainted json|error {
+    internal:JwtHeader header = {};
+    header.alg = JWT_HEADER_ALGO_VALUE;
+    header.typ = JWT_HEADER_TYPE_VALUE;
+
+    internal:JwtPayload payload = {};
+    payload.iss = serviceAccount;
+    payload.aud = [BASE_URL + TOKEN_ENDPOINT];
+
+    int iat = (time:currentTime().time / 1000) - 60;
+    int exp = iat + 3600;
+    payload.exp = exp;
+    payload.iat = iat;
+
+    map<any> customClaims = {};
```

Comment:
What are the types of values expected to be put in this map? Is `any` constraint really necessary?